src/middlewared/middlewared/api/v25_04_0/virt_volume.py

```diff
@@ -14,6 +16,18 @@
 ]
 
 
+RE_VOLUME_NAME = re.compile(r'^[A-Za-z][A-Za-z0-9-_.]*[A-Za-z0-9]$')
+
+
+def volume_name_validator(value: str) -> str:
```

Comment:
We actually already have a `match_validator` function that can be used instead of writing our own here. Please use that instead.